src/components/common/RadioGroup/RadioGroup.tsx

```diff
@@ -1,57 +1,95 @@
-import React, {useState, useEffect} from 'react'
+import Tippy from '@tippyjs/react'
+import React, { useState, useEffect } from 'react'
 import './radioGroup.scss'
 
 const RadioContext = React.createContext(null)
 
 function useRadioContext() {
     const context = React.useContext(RadioContext)
     if (!context) {
-        throw new Error(
-            `Radio compound components cannot be rendered outside the Toggle component`,
-        )
+        throw new Error(`Radio compound components cannot be rendered outside the Toggle component`)
     }
     return context
 }
 
-interface RadioGroupInterface{
-    name:string;
-    onChange:any;
-    className?:string;
-    initialTab:string;
-    disabled:boolean;
+interface RadioGroupInterface {
+    name: string
+    onChange: any
+    className?: string
+    initialTab: string
+    disabled: boolean
+    showTippy?: boolean
+    tippyContent?: any
 }
 
 interface RadioGroupComposition {
-    Radio?: React.FC<any>;
+    Radio?: React.FC<any>
 }
 
-const RadioGroup: React.FC<RadioGroupInterface> & RadioGroupComposition = React.memo(function RadioGroup({ name, onChange, children, className = "", initialTab, disabled=false }) {
+const RadioGroup: React.FC<RadioGroupInterface> & RadioGroupComposition = React.memo(function RadioGroup({
+    name,
+    onChange,
+    children,
+    className = '',
+    initialTab,
+    disabled = false,
+}) {
     const [selected, select] = useState(null)
 
-    useEffect(()=>{
-        if(initialTab === selected) return
+    useEffect(() => {
+        if (initialTab === selected) return
         select(initialTab)
-    },[initialTab])
+    }, [initialTab])
 
     return (
         <RadioContext.Provider value={{ name, selected, select, disabled, onChange }}>
-            <div className={`${className} radio-group`}>
-                {children}
-            </div>
+            <div className={`${className} radio-group`}>{children}</div>
         </RadioContext.Provider>
     )
 })
 
-function Radio({ value, children, className = "" }) {
+function Radio({ value, children, className = '', showTippy = false, tippyContent = '', canSelect = true }) {
     const { name, selected, select, disabled, onChange } = useRadioContext()
-    return (
+    return showTippy ? (
+        <Tippy className="default-tt fixed-width-250" arrow={false} placement="bottom" content={tippyContent}>
```

Comment:
can we use 'ConditionalWrap' component here?

Reply:
Done.